src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -102,13 +106,7 @@ export class DictionaryConcepts extends Component {
     const filterType = name.split(',')[1];
     const { typeName, collectionName, searchInput } = this.state;
     if (type === 'checkbox' && filterType === 'source') {
-      this.props.filterBySource(
-        inputName,
-        this.state.type,
-        typeName,
-        collectionName,
-        searchInput,
-      );
+      this.props.filterBySource(inputName, this.state.type, typeName, collectionName, searchInput);
```

Comment:
Is this a formatting change?

Reply:
@dondrzzy yes, It's formatting change.

Reply:
These should be addressed in a different PR.

Reply:
@dondrzzy   I have reverted all the formatting changes.